src/supervisor.adoc

```diff
@@ -2270,6 +2282,18 @@ Invalid PTEs using a bounded timer, or making address-translation caches
 coherent with store instructions that modify PTEs.
 ====
 
+[[sec:svkt]]
+== "Svkt" Extension for Address-Independent Latency of User-Mode Translation of Supervisor Addresses, Version 0.2
+
+The Svkt extension provides a means to make user-mode accesses to supervisor
```

Comment:
Naming suggestion: it's not the whole translation that occurs with constant time, just the exceptions.
```suggestion
The Svkte extension provides a means to make user-mode accesses to supervisor
```

Reply:
Changing the name is a bit of a pain, so I'm going to wait to see if there are further suggestions, then pick the best.